modules/renter/repair.go

```diff
@@ -49,9 +50,13 @@ func (f *file) repair(chunkIndex uint64, missingPieces []uint64, r io.ReaderAt,
 	var wg sync.WaitGroup
 	wg.Add(numPieces)
 	for i := 0; i < numPieces; i++ {
-		go func(host hostdb.Uploader, pieceIndex uint64, piece []byte) {
+		go func(i int) {
```

Comment:
do you need to pass in 'i' as an arg because this is a concurrent function? I guess so. But does that mean that the rest of the stuff is safe?

I guess it looks good to me, just want you to double check the correctness here.